zebra/zebra_evpn_mh.c

```diff
@@ -731,6 +732,29 @@ static void zebra_evpn_acc_bd_evpn_set(struct zebra_evpn_access_bd *acc_bd,
 	}
 }
 
+/* Lookup API for  VxLAN_IF's Bridge, VLAN in EVPN cache */
+int zebra_evpn_vl_vxl_bridge_lookup(uint16_t vid, struct zebra_if *vxlan_zif)
```

Comment:
Isn't it enough to use bool as a return type here?

Reply:
No really. There are three conditions, input variables are null then return -1, if value is not found then 0 and value is present then 1.